types/block.go

```diff
@@ -91,6 +92,9 @@ func (b *Block) ValidateBasic(chainID string, lastBlockHeight int64,
 	if !bytes.Equal(b.AppHash, appHash) {
 		return errors.New(cmn.Fmt("Wrong Block.Header.AppHash.  Expected %X, got %v", appHash, b.AppHash))
 	}
+	if !bytes.Equal(b.ConsensusHash, consensusHash) {
+		return errors.New(cmn.Fmt("Wrong Block.Header.ConsensusHash.  Expected %X, got %v", consensusHash, b.ConsensusHash))
```

Comment:
cmn.Fmt is deprecated, in the future I think just `fmt.Sprintf` would suffice ;)